Open the Cloud Editor - [HC-1553] (#361)
arduino/help-center-content: content/Arduino Cloud/Cloud Editor/Open-the-Cloud-Editor.md

---
title: "Open the Cloud Editor"
---

> 
> ## Open a sketch
> 

Reviewer: If the suggestion for line 15-25 is applied, this heading becomes redundant.
```suggestion

```

To open a sketch in the Cloud Editor, follow these steps:

1. Go to [app.arduino.cc/sketches](app.arduino.cc/sketches).

> 1. Click *Create* > *New Sketch* in the top-right.

Reviewer: It occurred to me that we can also include the option to open a saved sketch here:

```suggestion
1. Choose a sketch to open in the Cloud Editor:

    * **Create a new sketch:** Click on the Create button in the top-right corner and choose New Sketch.
    * **Open a saved sketch:** Click on a sketch you want to open.
```

Your sketch will automatically open in a new tab.

> ## Upload a sketch
> 
> To upload a sketch, you will need:
> 
> ### Windows, MacOS, or Linux
> 
> If you haven't already, install the [Arduino Create Agent](https://support.arduino.cc/hc/en-us/articles/360014869820-Install-the-Arduino-Create-Agent).
> 
> ### Chromebook
> 
> You will need [Arduino Cloud for Chromebook](https://support.arduino.cc/hc/en-us/articles/360016495639-Use-Arduino-with-Chromebook).

Reviewer: * Removed headings and reworked structure be more compact and "conclusive".
* MacOS → macOS
* Note: Added a heading id for the Chromebook article with 7c55771b6feb7a68d47aea28530a6657c5d76c42

```suggestion
To upload sketches to connected Arduino boards, please install the following software:

- **For Windows, macOS, or Linux:** [Install the Arduino Create Agent](https://support.arduino.cc/hc/en-us/articles/360014869820-Install-the-Arduino-Create-Agent).
- **For Chromebook:** [Install Arduino Cloud for Chromebook](https://support.arduino.cc/hc/en-us/articles/360016495639-Use-Arduino-with-Chromebook#install-arduino-cloud-for-chromebook).
```
